code/modules/mob/living/carbon/xenomorph/castes/sentinel/castedatum_sentinel.dm

```diff
@@ -37,13 +37,14 @@
 
 	// *** Ranged Attack *** //
 	spit_delay = 1.3 SECONDS
-	spit_types = list(/datum/ammo/xeno/toxin)
+	spit_types = list(/datum/ammo/xeno/toxin, /datum/ammo/xeno/acid)
```

Comment:
This needs to be replicated for all maturities.

Reply:
ok